openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -287,14 +288,32 @@ def __init__(
         # otherwise, we can't define two different ParameterHandler subclasses to compare for a new type of energy term
         # since both will try to register themselves for the same XML tag and an Exception will be raised.
         if parameter_handler_classes is None:
-            parameter_handler_classes = all_subclasses(ParameterHandler)
+            # TODO: What is the point of this argument? The default behavior includes a lot of logic.
+            #       Unaware of cases in which the argument is used.
```

Comment:
Let's keep this TODO in. I also don't know of anyone using a non-default argument here, but the behavior of `all_subclasses` is just so colossally automagic that I think there's decent value in letting users have access to a manual-entry mode.